src/platforms/react-native/manual-setup/hermes.mdx

```diff
@@ -88,6 +140,6 @@ Upload your source maps following [Step 3 on the normal source maps guide](/plat
 
 <Note>
 
-You will upload the **original** bundle (`.bundle`, output from Step 1) and the **composed** source map (`.map`, output from Step 3)
+You will upload the **original** bundle (`.bundle` or `.jsbundle`, output from Step 1) and the **composed** source map (`.map`, output from Step 3)
```

Comment:
It's not clear which steps you're referring to here. The three numbered steps in the previous section? If so, then suggest the following:
```suggestion
You will upload the **original** bundle (`.bundle` or `.jsbundle`, output from Step 1) and the **composed** source map (`.map`, output from Step 3) under [Compile Sourcemaps](#compile-sourcemaps).
```

if you mean the larger steps of this process, then we need to number the big headings and then make the steps under Compile Sourcemaps a, b, c, rather than 1, 2, 3.